src/Microsoft.DotNet.Interactive/Parsing/SubmissionParser.cs

```diff
@@ -555,6 +567,8 @@ private PolyglotParserConfiguration GetParserConfiguration()
         bool allowByRef,
         DirectiveExpressionNode expressionNode)
     {
+        // FIX: (RequestSingleValueFromKernel) make sure this is not dispatched on language service commands
```

Comment:
Did you intend to remove the `// FIX` comments before merge or is this something that needs to be fixed up in follow up PRs?

Reply:
I'll remove this one since it was resolved.